airborne_server/.env.example

```
@@ -1,6 +1,9 @@
 # Superposition configuration
 SUPERPOSITION_URL=http://localhost:8080
 SUPERPOSITION_ORG_ID=get-org-id-from-superposition
+SUPERPOSITION_CLEAR_UNUSED_PROVIDER=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPERPOSITION_CLEAR_UNUSED_PROVIDER key should go before the SUPERPOSITION_ORG_ID key

(UnorderedKey)